app/controllers/users_controller.rb

```diff
@@ -0,0 +1,55 @@
+class UsersController < ApplicationController
+  include ApplicationHelper
+
+  def index # login page
```

Comment:
This re-uses a standard action for something other than what a reader would expect.